common/app/experiments/Experiments.scala

```diff
@@ -64,3 +65,13 @@ object NewsletterEmbedDesign
       sellByDate = new LocalDate(2020, 11, 2),
       participationGroup = Perc0C,
     )
+
+object UseAusCmp
+    extends Experiment(
+      name = "use-aus-cmp",
+      description = "Use AU framework in CMP in Australia",
+      owners = group(Commercial),
+      sellByDate = new LocalDate(2020, 11, 6),
```

Comment:
How is this decided?

Reply:
i chose it, but it seems a good time away?